models/chef/Chef.validation.js

```diff
@@ -0,0 +1,18 @@
+import Joi from 'joi';
+import { accountStatus } from '../../constants/accountStatus';
+import { addressValidationSchema, phoneNumberPattern } from '../../helpers/validation';
+
+const ChefValidationSchema = Joi.object({
+  userId: Joi.required(),
+  avatar: Joi.string().required(),
+  phoneNumber: Joi.string().pattern(phoneNumberPattern).required(),
+  address: addressValidationSchema.required(),
+  certificate: Joi.string().required(),
+  accountStatus: Joi.string()
+    .valid(...accountStatus)
```

Comment:
It will be invalid joi schema: we should pass to `valid()` an array of simple values (`[1,2,3,'a']`) or an array of objects with value and message properties (`[ { value: 1, message: 'Should be 1!' } ]`)
https://joi.dev/api/?v=17.9.1#anyvalidvalues---aliases-equal